src/components/LinkChooser/utils/useManualComboBoxHandlers.ts

```diff
@@ -31,15 +31,18 @@ export const useManualComboBoxEventHandlers = (
             const item = state.collection.getItem(focusedKey);
             const { key } = event;
 
-            if (state.isOpen && SUBMISSION_KEYS.includes(key) && item) {
-                if (item.parentKey !== "search") {
+            if (state.isOpen && SUBMISSION_KEYS.includes(key)) {
+                if (item && item.parentKey !== "search") {
```

Comment:
```suggestion
                if (item?.parentKey !== "search") {
```

Reply:
This is not possible because if item is null then the if statement evaluates to true